libfwupdplugin/fu-config.c

```diff
@@ -104,6 +104,8 @@ fu_config_reload(FuConfig *self, GError **error)
 	/* ensure the config files are set to the correct permissions */
 	for (guint i = 0; i < priv->filenames->len; i++) {
 		const gchar *fn = g_ptr_array_index(priv->filenames, i);
+		if (g_str_has_prefix(fn, "/app"))
```

Comment:
We probably have the same problem for snap too? Maybe we should use environment variables instead.

Reply:
Can we ask the FS "is this immutable"? Is the chatter bit just set?

Reply:
Seems to work, will respin tomorrow.

Reply:
Done, all fixed up.